### ISSUE TITLE
stylelint-order not ordering `-` properly

### ISSUE THREAD

**Issue by thernstig** [stylelint/stylelint#8748]
### What minimal example or steps are needed to reproduce the bug?

```css
h1 {
  -webkit-font-smoothing: antialiased;
  text-rendering: optimizelegibility;
  -moz-osx-font-smoothing: grayscale;
}
```

### What minimal configuration is needed to reproduce the bug?

```js
// eslint-disable-next-line jsdoc/check-tag-names -- Needed since Stylelint does not support TypeScript
/** @type {import('stylelint').Config} */
export default {
  extends: ['stylelint-config-standard', 'stylelint-config-recess-order'],
  rules: {
    // Taken from https://github.com/hudochenkov/stylelint-config-hudochenkov/blob/master/order.js
    // Needed since stylelint-config-recess-order does not set the order/order property
    'order/order': [
      [
        'dollar-variables',
        'custom-properties',
        'at-rules',
        'declarations',
        {
          type: 'at-rule',
          name: 'supports',
        },
        {
          type: 'at-rule',
          name: 'starting-style',
        },
        {
          type: 'at-rule',
          name: 'container',
        },
        {
          type: 'at-rule',
          name: 'media',
        },
        'rules',
      ],
      { severity: 'warning' },
    ],
  },
  reportDescriptionlessDisables: true,
  reportInvalidScopeDisables: true,
  reportNeedlessDisables: true,
  reportUnscopedDisables: true,
};
```

### How did you run Stylelint?

`stylelint "**/*.css"`

### Which Stylelint-related dependencies are you using?

```json
    "stylelint": "^16.23.1",
    "stylelint-config-recess-order": "^7.2.0",
    "stylelint-config-standard": "^39.0.0",
    "stylelint-order": "^7.0.0",
```

### What did you expect to happen?

For it to order it as:

h1 {
  text-rendering: optimizelegibility;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
}

### What actually happened?

Nothing.

**Comment by stof** [comment]
this should probably be reported to https://github.com/hudochenkov/stylelint-order instead.

**Comment by thernstig** [comment]
@stof you are absolutely right. Done: https://github.com/hudochenkov/stylelint-order/issues/207.
